WordPress/gutenberg
# Delete experimental radix-ui based DropdownMenu component

Requires #55625

## What?


Remove the experimental, temporary version of `DropdownMenu` based off Radix UI

## Why?


The Radix UI version is being removed in favour of the ariakit-based version (see #50459 for more details)

## How?


- Delete all of the code related to the radix UI -based `DropdownMenuV2`
- Remove `radix-ui` as a dependency of the project
- Rename the ariakit based `DropdownMenuV2Ariakit` component to `DropdownMenuV2` (including folder structure).

**I recommend reviewing commit-by-commit.**

## Testing Instructions





- Make sure the project builds correctly
- Make sure that there are no traces left of Radix UI and the Radix UI-based DropdownMenu
- Make sure that the Storybook examples for the DropdownMenuV2 work as expected (ariakit implementation)
- Make sure that the DropdownMenus used in the dataviews package work as expected (see #55625 for more context)
